evm-adapters/src/lib.rs

```diff
@@ -222,6 +223,12 @@ pub trait Evm<State> {
 
     // TODO: Should we add a "deploy contract" function as well, or should we assume that
     // the EVM is instantiated with a DB that includes any needed contracts?
+
+    fn flatten_state(&self) -> HashSet<[u8; 32]>;
+}
+
+pub trait FuzzState {
+    fn flatten_state(&self) -> HashSet<[u8; 32]>;
 }
```

Comment:
is this needed in both places?

Reply:
one is a trait used for the `StackState`, and one is an impl on EVM that exposes it for us. Based on how sputnik works, i believe we do need both. when we move over to REVM shouldnt be necessary